packages/contracts-bedrock/src/L2/CrossL2Inbox.sol

```diff
@@ -65,13 +65,13 @@ contract CrossL2Inbox is ICrossL2Inbox, ISemver, TransientReentrancyAware {
     address internal constant DEPOSITOR_ACCOUNT = 0xDeaDDEaDDeAdDeAdDEAdDEaddeAddEAdDEAd0001;
 
     /// @notice Semantic version.
-    /// @custom:semver 1.0.0-beta.8
-    string public constant version = "1.0.0-beta.8";
+    /// @custom:semver 1.0.0-beta.9
+    string public constant version = "1.0.0-beta.9";
 
     /// @notice Emitted when a cross chain message is being executed.
     /// @param msgHash Hash of message payload being executed.
     /// @param id Encoded Identifier of the message.
-    event ExecutingMessage(bytes32 indexed msgHash, Identifier id);
+    event ExecutingMessage(bytes32 indexed msgHash, ICrossL2Inbox.Identifier id);
```

Comment:
We shouldn't import the interface within this file, instead we should re-define the struct within this contract.